### PR TITLE
🔧 try and fix path exclusion in CI

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

**Comment by oz123** [comment]
@Itxaka I reverted exclusion for `.github` paths too.

**Review by Itxaka — DISMISSED** [review]
looks good to me!

**Comment by Itxaka** [comment]
will this pick changes tot he files in the root dir? Like Earthfile and such?

**Comment by Itxaka** [comment]
I dont think so, maybe we need to also add `/*` ¬_¬

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -8,7 +8,13 @@ on:
...
     paths:
       - '!docs/**'
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
+      - 'images/**'
+      - 'overlay/**'
+      - 'skd/**'
```
Comment (mudler) [inline .github/workflows/image.yaml:17]:
```suggestion
      - 'sdk/**'
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -8,7 +8,13 @@ on:
...
     paths:
       - '!docs/**'
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
+      - 'images/**'
+      - 'overlay/**'
+      - 'skd/**'
```
Comment (mudler) [inline .github/workflows/lint.yml:17]:
```suggestion
      - 'sdk/**'
```

File: .github/workflows/unit-tests.yml [hunk 3]
```diff
@@ -8,7 +8,13 @@ on:
...
     paths:
       - '!docs/**'
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
+      - 'images/**'
+      - 'overlay/**'
+      - 'skd/**'
```
Comment (mudler) [inline .github/workflows/unit-tests.yml:17]:
```suggestion
      - 'sdk/**'
```

File: .github/workflows/unit-tests.yml [hunk 4]
```diff
@@ -8,7 +8,13 @@ on:
   pull_request:
     paths:
       - '!docs/**'
```
Comment (mudler) [inline .github/workflows/unit-tests.yml:10]: I guess we should include .github too?

File: .github/workflows/lint.yml [hunk 5]
```diff
@@ -8,7 +8,13 @@ on:
   pull_request:
     paths:
       - '!docs/**'
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
```
Comment (mudler) [inline .github/workflows/lint.yml:14]: ditto here

File: .github/workflows/unit-tests.yml [hunk 6]
```diff
@@ -8,7 +8,13 @@ on:
...
-      - '!.github/**'
+      - 'cmd/**'
+      - 'tests/**'
+      - 'pkg/**'
+      - 'internal/**'
```
Comment (mudler) [inline .github/workflows/unit-tests.yml:14]: would then skip if we touch the Earthfile?

File: .github/workflows/unit-tests.yml (hunk 4 above)
Comment (oz123) [reply .github/workflows/unit-tests.yml:10]: @Itxaka had concerns, so I removed it.

Comment (mudler) [reply .github/workflows/unit-tests.yml:10]: but if we don't include it among the `paths`, any modification to it won't trigger a workflow build, no?

**Review by mudler — COMMENTED** [review]
I think this approach isn't very flexible. Looking now at the PR here and how it grows makes it really error prone to maintain. Do we have also to include each file in the top root? I think if we can't make it _exclude_-only, that's going to bite us over and over again.

File: .github/workflows/unit-tests.yml (hunk 6 above)
Comment (mudler) [reply .github/workflows/unit-tests.yml:14]: I think that's the minimum requirement - everything we touch _except_ docs should trigger the CI. there are just few cases where we don't want them to trigger

Comment (oz123) [reply .github/workflows/unit-tests.yml:14]: That's what I tried to achieve with `!docs/**` according to GH documentation, it should work.
But should is a very rare fish :face_with_spiral_eyes:

**Review by mudler — APPROVED** [review]
Looks good! let's give it a try

**Pull request merged**